README.md

```diff
@@ -153,6 +153,7 @@ NOTE: You do not need to specify *end* segment - it will be added automatically.
 |--------|-------|-------|
 |`BULLETTRAIN_VIRTUALENV_BG`|`yellow`|Background color
 |`BULLETTRAIN_VIRTUALENV_FG`|`white`|Foreground color
+|`BULLETTRAIN_VIRTUALENV_NAME`|` %m`|Text to show after the prefix (use %m for virtualenv name)
```

Comment:
What if i use another prefix, and do not want to have a space between prefix and virtualenv name ?